src/python/WMCore/WMSpec/WMTask.py

```diff
@@ -10,6 +10,9 @@
 Equivalent of a WorkflowSpec in the ProdSystem.
 """
 
+from builtins import map, zip, str as str_, bytes
```

Comment:
Dario, to keep it consistent with the other slices, how about `str as newstr` instead?